apps/theme/components/Sidebar/ColorPane/ColorPane.module.css

```diff
@@ -62,6 +57,15 @@
   margin-top: -4px;
 }
 
+.cancel {
+  margin-right: auto;
+}
+
+.removeBtn {
+  margin-right: -16px;
+  margin-top: -4px;
```

Comment:
```suggestion
```
Makes the buttons not be aligned
![image](https://github.com/user-attachments/assets/029a8881-722d-41aa-b9e5-dff042fd228c)

If you want this, add it to `.cancel` as well